Modules/Utils.cs

```diff
@@ -1824,6 +1824,13 @@ static int GetInfoSize(string RoleInfo)
                         SelfName = GetString("DevouredName");
                 }
 
+                // Dollmaster, Prevent seeing self in mushroom cloud
+                if (CustomRoles.DollMaster.HasEnabled() && seerRole != CustomRoles.DollMaster)
+                {
+                    if (DollMaster.IsDoll(seer.PlayerId))
+                        SelfName = "<size=10000%><color=#000000>■</size></color>";
```

Comment:
html code is written incorrectly
Should be:
`"<size=10000%><color=#000000>■</color></size>"`

Reply:
I'll fix it, but how's that incorrect? I've always written html code like that in among us and never had an issue.

Reply:
In the provided snippet, it works as expected. it's just not good practice to begin closing the outer-most tags BEFORE closing the inner-most tags
![image](https://github.com/user-attachments/assets/7be4a74f-cca1-4a3a-923e-a3e86aeb6477)